### ISSUE TITLE
Accessible Design

### ISSUE THREAD

**Issue by studiomohawk** [frontendweekly/feeds#1397]
[Accessible Design](http://ift.tt/2gtXPB6) / Here at Sparkbox, we understand that accessibility is valuable to any engagement. The web is the future, and it should be available to any person regardless of&hellip;
